release_bot/init_repo.py

```diff
@@ -94,19 +104,36 @@ def create_conf(self, silent):
         cwd = os.getcwd()
         self.release_conf['author_email'] = run_command_get_output(cwd, f'git config user.email')[1].strip()
         self.release_conf['author_name'] = run_command_get_output(cwd, f'git config user.name')[1].strip()
+        if self.release_conf['author_email'] == "":
+            print("WARNING: your e-mail ID from git config is not set."+
+            	"\nPlease set it using \'git config user.email \"email@example.com\"\'")
```

Comment:
```suggestion
            	"\nPlease set it using 'git config user.email \"email@example.com\"'")
```

Reply:
`\'` is an escape sequence, to help Python print an ASCII `'` character.

However, simply using `'` in the print statement will cause a syntax error.

Reply:
not when you wrap the string with `"` as you do: the backslash is redundant:
```python
In [1]: print("\nPlease set it using 'git config user.email \"email@example.com\"'")  

Please set it using 'git config user.email "email@example.com"'


```

Reply:
Somehow pylint was showing this as a syntax error.

Verified, and done. 👍